indexer/src/types/transaction.ts

```diff
@@ -28,35 +26,69 @@ import {
   TxValuesArray,
   TypedTransaction,
   TypedTxData,
+  hexToBytes,
+  bytesToBigInt,
+  JsonTx,
 } from "../deps.ts";
 
+type HexString = `0x${string}`
+
+interface ToEthTxRequest {
+  transaction: Transaction;
+  receipt: TransactionReceipt;
+  blockNumber: PrefixedHexString;
+  blockHash: PrefixedHexString;
+  isPendingBlock: boolean;
+}
+
+interface ExtendedJsonRpcTx extends JsonRpcTx {
+  yParity?: string,
+  isRunOutOfResources?: boolean 
+}
+
+interface TypedTransactionToEthTxRequest {
```

Comment:
This naming seems too long and confusing to me, we should keep names as short as possible

Reply:
changed to "TypedTxToEthTx"